cmd/revendor/run-glide.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/Masterminds/glide/action"
+	"github.com/Masterminds/glide/repo"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	var (
+		pkg      string
+		revision string
+	)
+	flag.StringVar(&pkg, "package", "", "name of the vendored package to update")
+	flag.StringVar(&revision, "revision", "", "revision to which to update")
+	flag.Parse()
+	if pkg == "" || revision == "" {
+		grip.Info("no repo to revendor, run-glide will no-op")
+		return
+	}
+
+	updateGlide(pkg, revision)
+
+	installer := repo.NewInstaller()
+	action.EnsureGoVendor()
+	action.Install(installer, false, false)
+}
+
+func updateGlide(pkg, revision string) {
+	wd, err := os.Getwd()
+	if err != nil {
+		grip.Error(errors.Wrap(err, "error getting working dir"))
+		return
+	}
```

Comment:
is it ok to continue on error here, but not in any other situation?

Reply:
oops, fixed